app/models/digital_instantiation.rb

```diff
@@ -159,7 +159,4 @@ def save_instantiation_admin_data
     find_or_create_instantiation_admin_data
     self.instantiation_admin_data.save
   end
-
-  # This must be included at the end, because it finalizes the metadata if you have any further properties define above in current model
-  include ::Hyrax::BasicMetadata
```

Comment:
@orangewolf is the comment here no longer a concern?

Reply:
the one above says the same thing - it has to be after any properties. putting it under the private is the issue